<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camoufox Profile Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutLeft">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QListWidget" name="profileList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl_listButtons">
        <item>
         <widget class="QPushButton" name="newProfileButton">
          <property name="text">
           <string>New Profile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteProfileButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutRight">
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>24</number>
        </property>
        <property name="verticalSpacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>16</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelViewport">
          <property name="text">
           <string>Viewport</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="hl_viewport">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelW">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinW">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="minimum">
             <number>320</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1280</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>12</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelH">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinH">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="minimum">
             <number>320</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelFullscreen">
          <property name="text">
           <string>Fullscreen</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="fullscreenCheck">
          <property name="text">
           <string>Launch in fullscreen</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelProxy">
          <property name="text">
           <string>Proxy</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="proxyGroup">
          <property name="title">
           <string>Proxy (optional)</string>
          </property>
          <layout class="QGridLayout" name="gridLayoutProxy">
           <property name="horizontalSpacing">
            <number>16</number>
           </property>
           <property name="verticalSpacing">
            <number>12</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lblHost">
             <property name="text">
              <string>Host</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="proxyHostEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lblPort">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="proxyPortSpin">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblUser">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="proxyUserEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="lblPass">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="proxyPassEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="QCheckBox" name="geoipCheck">
             <property name="text">
              <string>Use GeoIP with Proxy</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="hl_storage">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QLineEdit" name="storageEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QPushButton" name="browseStorageButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Browse…</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelStorage">
          <property name="text">
           <string>Storage directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl_actions">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Save Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="actionSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="launchButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Launch Session</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Stop Session</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
